0003_saslauthd_mdoc.dpatch by  <fabbe@debian.org>

Use the correct path for the saslauthd.conf file, and use another
date format (cosmetic).

diff -urNad trunk~/saslauthd/saslauthd.mdoc trunk/saslauthd/saslauthd.mdoc
--- trunk~/saslauthd/saslauthd.mdoc	2006-05-29 22:52:42.000000000 +0300
+++ trunk/saslauthd/saslauthd.mdoc	2006-07-12 15:05:25.000000000 +0300
@@ -10,7 +10,7 @@
 .\" manpage in saslauthd.8 whenever you change this source
 .\" version. Only the pre-formatted manpage is installed.
 .\"
-.Dd 10 24 2002
+.Dd October 24 2002
 .Dt SASLAUTHD 8
 .Os "CMU-SASL"
 .Sh NAME
@@ -216,7 +216,7 @@
 .Em (All platforms that support OpenLDAP 2.0 or higher)
 .Pp
 Authenticate against an ldap server.  The ldap configuration parameters are
-read from /usr/local/etc/saslauthd.conf.  The location of this file can be
+read from /etc/saslauthd.conf.  The location of this file can be
 changed with the -O parameter. See the LDAP_SASLAUTHD file included with the
 distribution for the list of available parameters.
 .It Li sia
@@ -249,7 +249,7 @@
 .Bl -tag -width "/var/run/saslauthd/mux"
 .It Pa /var/run/saslauthd/mux
 The default communications socket.
-.It Pa /usr/local/etc/saslauthd.conf
+.It Pa /etc/saslauthd.conf
 The default configuration file for ldap support.
 .El
 .Sh SEE ALSO
